Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -213,6 +213,18 @@ public class JavaScriptLifter: Lifter {
                 }
                 output = ArrayLiteral.new("[" + elems.joined(separator: ",") + "]")
 
+            case let op as CreateTemplateLiteral:
+                var literals = [String]()
+                if op.literals.count > 0 {
```

Comment:
I think `assert(!op.parts.isEmpty)` would prevent empty templates `(e.g. let v1 = ``)`.

Reply:
See comment below, I think we can just enforce that there is at least one part, but that part can be the empty string, so that'd give you an empty literal. WDYT?

Reply:
Ah nice, yeah that would work :+1: